Dockerfile

```diff
@@ -1,24 +1,29 @@
-FROM ubuntu:bionic
+FROM ubuntu:focal
 LABEL author="artur@barichello.me"
 
 USER root
-
-RUN apt-get update && apt-get install -y --no-install-recommends \
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get full-upgrade --yes && apt-get install --yes --no-install-recommends \
     ca-certificates \
     git \
+    git-lfs \
     python \
     python-openssl \
     unzip \
     wget \
     zip \
     adb \
     openjdk-8-jdk-headless \
+    imagemagick \
+    wine-stable \
     && rm -rf /var/lib/apt/lists/*
 
+RUN wget --quiet https://github.com/electron/rcedit/releases/download/v1.1.1/rcedit-x64.exe -O /usr/local/bin/rcedit.exe
+
 ENV GODOT_VERSION "3.2.3"
 
-RUN wget https://downloads.tuxfamily.org/godotengine/${GODOT_VERSION}/Godot_v${GODOT_VERSION}-stable_linux_headless.64.zip \
-    && wget https://downloads.tuxfamily.org/godotengine/${GODOT_VERSION}/Godot_v${GODOT_VERSION}-stable_export_templates.tpz \
+RUN wget --quiet https://downloads.tuxfamily.org/godotengine/${GODOT_VERSION}/Godot_v${GODOT_VERSION}-stable_linux_headless.64.zip \
```

Comment:
I like this "spam in the build log", it helps make sure the download path didn't break upstream. Does it still show errors with the quiet flag? If so then it's fine.

Reply:
`curl -fsSL` is typically what I use with curl (quiet normally, but displays errors and follows redirects).

Reply:
After checking an observing that `--quiet` does indeed suppress, for example, 404s, I've changed it to `--no-verbose`. I agree this is much better functionality.